Cargo.toml

```diff
@@ -0,0 +1,8 @@
+[package]
+edition = "2021"
+name = "big-int"
+version = "0.1.0"
+[dependencies]
+eyre = "*"
```

Comment:
also, as this crate is a library, it is better to use `thiserror` instead (and keep `eyre` only for binary runtime)

Reply:
Okay, I didn't know about `thiserror`, I will have a look at it